influxdb/00-init.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
I'm using various bash-specific options for shell script robustness:
```
set -o errexit
set -o nounset
set -o pipefail
set -o errtrace
```
They aren't available in regular sh.

Reply:
Okk gotcha.